tests/integration/test_store_synapse.py

```diff
@@ -0,0 +1,125 @@
+from unittest.mock import MagicMock
+
+import numpy as np
+import pytest
+
+from schematic.schemas.data_model_graph import DataModelGraphExplorer
+from schematic.store.synapse import SynapseStorage
+from schematic.utils.validate_utils import comma_separated_list_regex
+from tests.conftest import Helpers
+
+
+@pytest.fixture(name="dmge", scope="function")
+def DMGE(helpers: Helpers) -> DataModelGraphExplorer:
+    """Fixture to instantiate a DataModelGraphExplorer object."""
+    dmge = helpers.get_data_model_graph_explorer(path="example.model.jsonld")
+    return dmge
```

Comment:
This does not need to be copied into individual files. I have included this code within the `conftest.py` file

Reply:
Thanks Bryan. I will remove it.